tests/sqlparser_postgres.rs

```diff
@@ -5098,3 +5098,34 @@ fn parse_create_type_as_enum() {
         _ => unreachable!(),
     }
 }
+
+#[test]
+fn parse_bitstring_literal_in_projection_is_not_alias() {
+    let select = pg_and_generic().verified_only_select("SELECT B'111'");
+    assert_eq!(
+        select.projection,
+        vec![SelectItem::UnnamedExpr(Expr::Value(
+            Value::SingleQuotedByteStringLiteral("111".to_string())
+        ))]
+    );
+}
+
+#[test]
+fn parse_bitstring_literal_in_insert_value_succeeds() {
+    let stmt = pg_and_generic().verified_stmt("INSERT INTO foo VALUES (B'111')");
+    match &stmt {
+        Statement::Insert(Insert {
+            source: Some(query),
+            ..
+        }) => assert_eq!(
+            *query.body,
+            SetExpr::Values(Values {
+                explicit_row: false,
+                rows: vec![vec![Expr::Value(Value::SingleQuotedByteStringLiteral(
+                    "111".to_string()
+                ))]]
+            })
+        ),
+        _ => unreachable!(),
+    }
+}
```

Comment:
```suggestion
#[test]
fn parse_bitstring_literal() {
    let select = pg_and_generic().verified_only_select("SELECT B'111'");
    assert_eq!(
        select.projection,
        vec![SelectItem::UnnamedExpr(Expr::Value(
            Value::SingleQuotedByteStringLiteral("111".to_string())
        ))]
    );
}
```
Thinking its probably enough to show that the dialect is enabled to parse bitstrings, and that its not specific to select/insert. (Same fore mysql)

Reply:
Fair enough!